content/en/profiler/enabling/java.md

```diff
@@ -131,7 +132,22 @@ java \
 {{% /tab %}}
 {{< /tabs >}}
 
-    **Note**: The `-javaagent` argument needs to be before `-jar`, adding it as a JVM option rather than an application argument. For more information, see the [Oracle documentation][6]:
+{{% collapse-content title="Build and run native-image" level="h4" %}}
+
+Follow the [Tracer Setup Instructions][13] to build your native image with the Datadog Java Profiler.
+
+When the service binary is built you can use environment variables to enable and configure the Datadog Java Profiler.
+
+   ```shell
+   DD_PROFILING_ENABLED=true DD_PROFILING_DIRECTALLOCATION_ENABLED=true ./my_service
+   ```
+
+   **Note**: Only JFR based profiling is supported for the GraalVM native-image applications. None of the 'DDPROF'
+related configuration options is effective.
+
+{{% /collapse-content %}}
+
+   **Note**: The `-javaagent` argument needs to be before `-jar`, adding it as a JVM option rather than an application argument. For more information, see the [Oracle documentation][6]:
 
     ```shell
```

Comment:
This code block isn't rendering properly in the preview anymore. Deleting one space before each line should fix it. Side note: if we change the code fence to `bash` instead of `shell` it won't include the `$` on the first line.